management/server/account.go

```diff
@@ -738,18 +741,19 @@ func (a *Account) UserGroupsRemoveFromPeers(userID string, groups ...string) {
 
 // BuildManager creates a new DefaultAccountManager with a provided Store
 func BuildManager(store Store, peersUpdateManager *PeersUpdateManager, idpManager idp.Manager,
-	singleAccountModeDomain string, dnsDomain string, eventStore activity.Store,
+	singleAccountModeDomain string, dnsDomain string, eventStore activity.Store, userDeleteFromIDPEnabled bool,
```

Comment:
Yes you are right but what can I proof with it? We can do test to verify to call the external idpManager.DeleteUser() or not. But in this case the idpManager could be a moc interface so at the end we will verify that the moc implementation works fine or not.
But I am not a test expert. If you think it have any benefits in unit test layer then I will write tests for it.

Reply:
Enabling the `userDeleteFromIDPEnabled=true` testing option will exclusively assess the mock scenario and won't involve actual deletion through the IDP API. We can address this by conducting manual testing for each IDP with the flag enabled.

Reply:
Yes, more likely it will be just checking if the correct method is being called and certain error paths. But you might be right there are not many benefits in it currently.